.github/workflows/terraform-zones-deploy.yml

```diff
@@ -8,8 +8,7 @@ on:
       - 'main'
     paths:
       - 'terraform/**'
-      - '.github/workflows/terraform-**'
-
+      - '.github/workflows/terraform-*.yml'
```

Comment:
this line looks like it's being remerged from `main`, which was ahead of `dev`?

Reply:
i was trying to piece that together but it looks like the change was still present in dev already?